src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/CpsPackageReferenceProject.cs

```diff
@@ -36,18 +35,14 @@ namespace NuGet.PackageManagement.VisualStudio
     /// Key feature/difference is the project restore info is pushed by nomination API and stored in 
     /// a cache. Factory method retrieving the info from the cache should be provided.
     /// </summary>
-    public class CpsPackageReferenceProject : BuildIntegratedNuGetProject
+    public class CpsPackageReferenceProject : PackageReferenceProject
     {
         private const string TargetFrameworkCondition = "TargetFramework";
 
-        private readonly string _projectName;
-        private readonly string _projectUniqueName;
-        private readonly string _projectFullPath;
-
         private readonly IProjectSystemCache _projectSystemCache;
         private readonly UnconfiguredProject _unconfiguredProject;
-        private List<(NuGetFramework, Dictionary<string, ProjectInstalledPackage>)> _installedPackages = new List<(NuGetFramework, Dictionary<string, ProjectInstalledPackage>)>();
-        private DateTime _lastTimeAssetsModified;
+        private List<(NuGetFramework targetFramework, Dictionary<string, ProjectInstalledPackage> packages)> _installedPackages = new List<(NuGetFramework, Dictionary<string, ProjectInstalledPackage>)>();
```

Comment:
I love that you've replaced the unhelpful default tuple names (e.g.:  `Item1`) with more meaningful names, but the names should be capitalized (like a property).  Otherwise, it [looks abnormal](https://github.com/NuGet/NuGet.Client/pull/3680/files#diff-5fae96e8312674fe53149dbdb5a5f036aea82698a5aeff49617e8e7ae66d2685R273-R274).

The feedback applies here and elsewhere.

Reply:
Fixed